src/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -6,8 +6,6 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <ContractProject>..\ref\System.ComponentModel.Composition.depproj</ContractProject>
-    <UseEcmaKey>true</UseEcmaKey>
```

Comment:
I removed it because the reference assembly didnt have it and they have to match, but I can add it in both.